ophyd/pseudopos.py

```diff
@@ -126,11 +334,14 @@ class PseudoPositioner(Device, Positioner):
     '''
     def __init__(self, prefix, *, concurrent=True, read_attrs=None,
                  configuration_attrs=None, monitor_attrs=None, name=None,
-                 **kwargs):
+                 egu='', **kwargs):
```

Comment:
how do we want to deal with mixed unit positions, ex '(rad, mm, mm)'  (theta, r, z) in cylindrical cord.  Is that a 'later' problem?